data/language/en-GB.txt

```diff
@@ -3642,6 +3642,8 @@ STR_6536    :This park was saved in a later version of OpenRCT2. The park was sa
 STR_6537    :Allow using regular paths as queue
 STR_6538    :Shows regular paths in the queues dropdown of the Footpaths window.
 STR_6539    :Brake Closed
+STR_6540    :{WINDOW_COLOUR_2}A Special Thanks to the following companies for their permission:
```

Comment:
While I realise this sentence was likely shortened for brevity, I think we clarify what the permission was _for_.

Reply:
Perhaps `Special thanks to the following companies for allowing their likeness:`?